src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Figure implements AreaCalculation {
+
+    private String color;
+    private String name;
+
+    public Figure() {
+    }
+
+    public Figure(String color, String name) {
+        this.color = color;
+        this.name = name;
+    }
+
+    public String printInfo() {
+        return "Figure: "
+                + name
+                + " area: "
+                + getArea()
+                + " sq.units"
+                + " color: "
+                + color;
+    }
+
+    @Override
+    public double getArea() {
+        return 0;
+    }
```

Comment:
dont implement those methods here
you should implement it in classes Circle, Square etc

Reply:
not fixed